sdk/cdsclient/interface.go

```diff
@@ -14,6 +14,17 @@ type Filter struct {
 	Name, Value string
 }
 
+// TemplateClient exposes templates functions
+type TemplateClient interface {
+	TemplateGet(groupName, templateSlug string) (*sdk.WorkflowTemplate, error)
+	TemplateGetByID(id int64) (*sdk.WorkflowTemplate, error)
+	TemplateGetAll() ([]*sdk.WorkflowTemplate, error)
+	TemplateApply(groupName, templateSlug string, req sdk.WorkflowTemplateRequest) (*tar.Reader, error)
+	TemplatePull(groupName, templateSlug string) (*tar.Reader, error)
+	TemplatePush(tarContent io.Reader) ([]string, *tar.Reader, error)
+	TemplateGetInstances(groupName, templateSlug string) ([]*sdk.WorkflowTemplateInstance, error)
```

Comment:
[]*sdk.WorkflowTemplateInstance is not good

Reply:
done